capstone/src/main/resources/static/event.js

```diff
@@ -66,93 +116,225 @@ function createEventElement(eventListElement, event, displaySaveButton, displayU
   createElement(eventElement, 'p', event.eventLatitude);
 
   // Organization
-  //const eventOrgElement = document.createElement('p');
-  //eventOrgElement.innerText = event.organization.name;
-  //eventElement.appendChild(eventOrgElement);
-
-  // Displays only for individual users
-  // create save, unsave, delete, or edit event form
-    if (displayEditDeleteButtons) {
-      // if edit is allowed, then it means that delete is allowed as well
-      createEditAndDeleteEventButton(eventElement, event);
-    } else if (displaySaveButton) {
-      createSaveEventButton(eventElement, event);
-    } else if (displayUnsaveButton) {
+  createElement(eventElement, 'p', event.organizationName);
+
+  // Only for individual users can save/unsave events
+  if (isIndividual) {
+    if (userSavedEvent) { // Individual has saved event
       createUnsaveEventButton(eventElement, event);
+    } else {
+      createSaveEventButton(eventElement, event);
     }
-    return eventElement;
+  } else {
+    createEditAndDeleteButton(eventElement, event);
```

Comment:
I think this should be createEditAndDeleteEventButton. It is throwing an error in the console